netbox/users/admin.py

```diff
@@ -16,14 +35,26 @@ class UserConfigInline(admin.TabularInline):
     verbose_name = 'Preferences'
 
 
-@admin.register(User)
+@admin.register(AdminUser)
 class UserAdmin(UserAdmin_):
     list_display = [
         'username', 'email', 'first_name', 'last_name', 'is_superuser', 'is_staff', 'is_active'
     ]
+    fieldsets = (
+        (None, {'fields': ('username', 'password')}),
+        ('Personal info', {'fields': ('first_name', 'last_name', 'email')}),
+        ('Permissions', {
+            'fields': ('is_active', 'is_staff', 'is_superuser'),
```

Comment:
I think groups need to be added.
`            'fields': ('is_active', 'is_staff', 'is_superuser', 'groups'),`

Reply:
This is still a WIP and not ready for review.